internal/pubkey/encryption_types.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 Finobo
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pubkey
+
+import (
+	"fmt"
+
+	"github.com/mailchain/mailchain/crypto"
+)
+
+const (
```

Comment:
These constants already exist in `crypto/cipher/encrypter/encrypter.go`, can you use these constants instead. Can you copy over your comments 😄

Reply:
Thanks for pointing that out 😄 , didn't notice them.